src/Themes.scss

```diff
@@ -825,6 +825,18 @@ body.gray-theme {
         border-right: 1px solid #9DA4AF;
     }
 
+    .ProgramBlockEditor__character-position-button--disabled:not(.ProgramBlockEditor__character-turn-positions) {
```

Comment:
I believe that the `:not(.ProgramBlockEditor__character-turn-positions)` selector will have no effect here, as the `ProgramBlockEditor__character-turn-positions` class would be on the parent element of `ProgramBlockEditor__character-position-button--disabled`, rather than the same element.

There's another similar rule below, on line 1408, for the high contrast theme.